### PR TITLE
{bio}[foss/2019b] FusionCatcher v1.30 w/ Python 2.7.16

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3501.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/2da58af4dde1532c0fd845861c93d625 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3122.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/4a5e79235b58ad27c6237642bad8310d for a full test report.

**Comment by lexming** [comment]
The dev updated to v1.30 without a release or even a tag? Seriously? I'm in awe...

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 7 out of 7 (1 easyconfigs in total)
node353.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/470710fdba988a9db1fe626d8527cd30 for a full test report.

**Comment by lexming** [comment]
Running the extended tests...

**Comment by boegel** [comment]
> The dev updated to v1.30 without a release or even a tag? Seriously? I'm in awe...

https://github.com/ndaniel/fusioncatcher/issues/179

**Comment by boegel** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/f/FusionCatcher/FusionCatcher-1.30-foss-2019b-Python-2.7.16.eb [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+default_component_specs = {
+    'sources': [{'download_filename': '%(version)s.tar.gz', 'filename': SOURCE_TAR_GZ}],
+    'start_dir': '%(namelower)s-%(version)s',
+}
+
+components = [
+    (name, version, {
+        'easyblock': 'Tarball',
+        'source_urls': [GITHUB_SOURCE],
+        'checksums': ['a6313ebee6f7225fa873d901e4c9ff7e0990763cde754df8db40d10523a69800'],
```
Comment (verdurin) [inline easybuild/easyconfigs/f/FusionCatcher/FusionCatcher-1.30-foss-2019b-Python-2.7.16.eb:47]: @boegel checksum needs to be updated?

**Review by verdurin — DISMISSED** [review]

**Comment by lexming** [comment]
The tests in FusionCatcher (`$EBROOTFUSIONCATCHER/test/test.sh`) were all OK:
https://gist.github.com/lexming/a143ef3df8cf90341f291b13ff0adf1b
(done with 4 cores and 24GB of RAM)

File: easybuild/easyconfigs/f/FusionCatcher/FusionCatcher-1.30-foss-2019b-Python-2.7.16.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/f/FusionCatcher/FusionCatcher-1.30-foss-2019b-Python-2.7.16.eb:47]: Ah, yes, the name of the directory in the source tarball changed when switching to the properly tagged release...

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2602.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/3c6967893604e6a70c33851a97b2e405 for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node381.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/b26fd22337f43c212ef3edec6b962df2 for a full test report.

**Comment by lexming** [comment]
Going in, thanks @boegel !

**Pull request merged**